Automattic/jetpack
# Fix Network Admin screen connection status when Jetpack is installed in MU-Plugins

Fixes #13478

#### Changes proposed in this Pull Request:
* Add a check for the plugin located in mu-plugins

#### Other information:


## Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Open up a WordPress Multisite
* Go to `wp-admin/network/admin.php?page=jetpack` 
* Verify that the "Connected" Column correctly displays the connection status of the network sites
<img width="717" alt="Screen Shot 2022-07-18 at 1 41 30 PM" src="https://user-images.githubusercontent.com/3220162/179613705-08e67800-f469-49f5-9df5-b94df0a37dbb.png">
